src/a5dd0_len_114e0.c

```diff
@@ -863,10 +856,23 @@ INCLUDE_ASM(s32, "a5dd0_len_114e0", create_shadow_from_data);
 
 INCLUDE_ASM(s32, "a5dd0_len_114e0", MakeEntity, ScriptInstance* script, s32 isInitialCall);
 
-
 INCLUDE_ASM(s32, "a5dd0_len_114e0", SetEntityCullMode);
 
-INCLUDE_ASM(s32, "a5dd0_len_114e0", UseDynamicShadow);
+ApiStatus UseDynamicShadow(ScriptInstance* script, s32 isInitialCall) {
+    Entity* entity = get_entity_by_index(gLastCreatedEntityIndex);
+
+    if (get_variable(script, *script->ptrReadPos) != 0) {
```

Comment:
The variable here is a bool `useDynamicShadow?`, so just do
```suggestion
    if (get_variable(script, *script->ptrReadPos)) {
```